src/AggregateFunctions/AggregateFunctionSparkbar.h

```diff
@@ -104,14 +105,14 @@ class AggregateFunctionSparkbar final
     X min_x;
     X max_x;
     bool specified_min_max_x;
-    using Chars = PODArray<char>;
 
     template <class T>
-    void updateFrame(Chars & frame, const T value) const
+    int updateFrame(ColumnString::Chars & frame, const T value) const
     {
         static const String bars[9] = {" ", "▁", "▂", "▃", "▄", "▅", "▆", "▇", "█"};
         const auto & bar = (isNaN(value) || value > 8 || value < 1) ? bars[0] : bars[static_cast<UInt8>(value)];
         frame.insert(bar.begin(), bar.end());
+        return static_cast<int>(bar.size());
```

Comment:
why it should be an int?

Reply:
`bar` values are limited to 9 values, which is quite small, so I think using `int` is enough.

Reply:
But it also cannot be negative - so the type should be unsigned.